ScottyLabs/cmu_lost_and_found_v2
# Add an “Archived” tab to the admin page (#158)

# Description

This PR adds a switch to the admin panel that allows us to choose between active (which is the current behavior) and archived items.

<img width="1362" alt="Screenshot 2022-09-24 at 16 37 54" src="https://user-images.githubusercontent.com/7029582/192118216-af6f17d0-3e0d-4b15-8ac7-a984c40503a4.png">


Closes #158

## Type of change


# How Has This Been Tested?

1. Log in and go to the admin panel
2. Expect to see the non-archived items
3. Click on “Archived”
4. Expect the archived items

**Test Configuration**:

- Desktop/Mobile: Desktop
- OS: macOS
- Browser: Firefox 104, Safari 15

# Checklist:

- ~~I have made corresponding changes to the documentation~~ (not applicable)
- ~~I have added tests that prove my fix is effective or that my feature works~~ (not applicable)
- ~~New and existing unit tests pass locally with my changes~~ (not applicable)
